book/4-end/.gitignore

```diff
@@ -11,5 +11,4 @@ npm-debug.log
 node_modules/
 .coverage
 .env
-.next
-package-lock.json
```

Comment:
oh noes, accidentally committed this.
package-lock.json was generated by npm (I use it instead of yarn) - so I added it to ignore list, but forgot that .gitignore is also tracked by git.